package.json

```diff
@@ -11,8 +11,13 @@
     "dev:remix": "cross-env NODE_ENV=development netlify dev",
     "generate:css": "npx tailwindcss -o ./app/tailwind.css",
     "start": "cross-env NODE_ENV=production netlify dev",
+    "lint": "eslint \"./{app,studio}/**/*.{ts,tsx,js,jsx}\"",
+    "lint:fix": "npm run lint -- --fix",
     "sanity-codegen": "sanity-codegen"
   },
+  "prettier": {
+    "trailingComma": "all"
```

Comment:
🤷 👍  Jeg er ganske nøytral på denne, det som er bra med prettier er at jeg slipper å forholde meg til det (gitt at autoformat fungerer, som jeg opplever det gjør for meg)